### ISSUE TITLE
Optimize live announcement from carousel

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#4547]
### Scope

The carousel uses an aria-live region to announce new slides if they are in view.
1. This doesn't work in Chrome/VO if the user uses the arrow buttons
2. On Safari/VO all contents of all visible slides are announenced which is distracting
3. If there are UI elements inside a carousel slide which the user can use (e.g. toggle a whishlist button) then all contents of the slides get announced.
4. Slide status announcement should be exposed as well

### Notes

You can reproduce the issue on the PDS UI Testing page: 
https://porsche-design-system.github.io/examples/pr-226/pds-ui-testing/en/

### Acceptance criteria

- [ ] Optimize carousel screen reader announcements
